draft-ietf-quic-transport.md

```diff
@@ -3804,6 +3804,13 @@ number length, connection ID length, and path MTU.  A receiver MUST accept
 packets containing an outdated frame, such as a MAX_DATA frame carrying a
 smaller maximum data than one found in an older packet.
 
+A sender SHOULD avoid retransmitting information from packets once they are
+acknowledged. This includes packets that are acknowledged after being declared
+lost. Doing so requires senders to retain information about packets after they
+are declared lost. A sender can discard this information after a period of time
+elapses, such as a PTO (Section 6.2 of {{QUIC-RECOVERY}}), or on other events,
+such as reaching a memory limit.
```

Comment:
```suggestion
acknowledged. Doing this for packets acknowledged after they are declared
lost requires retaining state for lost packets. A sender can discard this information
after the expected maximum packet reordering elapses, such as after PTO has elapsed
(Section 6.2 of {{QUIC-RECOVERY}}), or on other events, such as reaching a packet limit.
```

Reply:
I will take the compression, but we don't need to say "expected maximum packet reordering", since that's not a well-defined term. I'll spell in out in other words.

Reply:
I'm trying to wordsmith this compression, but it's a bit difficult. The text is a bit verbose, but I think it is clearer. Unless you think there's a substantive difference, I'd prefer the existing text for clarity.

Reply:
The expanded form is fine.

Reply:
The intent was to both make it clearer and talk about the goal of waiting for a reordering period.  I find the existing text a bit unclear, which is why I put the first two sentences together.

Reply:
Ok, added some rationale.